src/lib/utils/env.js

```diff
@@ -0,0 +1,12 @@
+import config from '%src/config'
+import constants from '%src/constants/env'
```

Comment:
calling this constants feels off to me (similar to calling utils/env `utils` in the earlier method), but I appreciate my gut might be wrong here...

What would you do if you needed to import constants from more than one source?

What if instead you did:

```
import { ENV_NAMES } from `%src/constants/env`
```

Reply:
Thinking more about this (and the utils parallel) I do think we should default to pulling the specific items we're using.  Spares us the naming question and also helps us identify dead code some day down the line.

Reply:
Ditto.